lib/govuk_app_config/govuk_error/configure.rb

```diff
@@ -1,11 +1,13 @@
 GovukError.configure do |config|
+  config.current_environment = ENV["SENTRY_CURRENT_ENV"]
```

Comment:
Is this needed?

Reply:
Without it, I get a failing test:

```
Failures:

  1) Requiring govuk_app_config correctly initialises error tracking
     Failure/Error: expect(Raven.configuration.current_environment).to eql("integration-or-somesuch")
     
       expected: "integration-or-somesuch"
            got: "default"
     
       (compared using eql?)
     # ./spec/integration/configuring_spec.rb:8:in `block (3 levels) in <top (required)>'
     # ./spec/integration/configuring_spec.rb:5:in `block (2 levels) in <top (required)>'
```

Reply:
Hmm the curious thing to me is that we weren't setting current environment before and I assume this already worked? Is the failing test due to Raven configuration being a singleton and maintaining state?